src/cuda/plugin-PixelTriplets/BrokenLineFitOnGPU.cu

```diff
@@ -18,13 +15,31 @@ void HelixFitOnGPU::launchBrokenLineKernels(HitsView const *hv,
   auto fast_fit_resultsGPU_ = cms::cuda::make_device_unique<double[]>(
       maxNumberOfConcurrentFits_ * sizeof(Rfit::Vector4d) / sizeof(double), stream);
 
+  cms::cuda::ExecutionConfiguration exec;
```

Comment:
Could all these be done in the `cudadev` program instead (or "fork" a separate program, e.g. `cudaautotune`)? I'd like to keep the reference `cuda` program as close to the original as possible.

Reply:
I forked it to cudaautotune, and changed the tuning script directory to be under `src/cudaautotune/autotuining`.